### DISCUSSION TITLE
Does this work with Candy CSOE H8A3TE-S?

### DISCUSSION THREAD

**Discussion by quack3d** [ofalvai/home-assistant-candy#118] [Device support improvements]
Can someone confirm this works with the tumble dryer Candy CSOE H8A3TE-S? If it does I'll order straight away. :)

https://www.candy-home.com/da_DK/t-rretumlere/31102177/csoe-h8a3te-s/

**Comment by ofalvai** [comment]
Hard to say for sure, but I see the simply-Fi app mentioned. This integration uses the same APIs as that app, so it will probably work with the above device.

**Reply by quack3d** [reply]
Thanks. It also says hOn app on product sheet. I gambled and ordered one yesterday anyways.

**Comment by quack3d** [comment]
It wouldn't be added to the simply-Fi app unfortunately. Have to use hOn. :(

**Comment by FilipeOS** [comment]
Does this work with Candy CSO H8A2TE-S?
